tests/gha-run-k8s-common.sh

```diff
@@ -31,12 +31,16 @@ function _print_instance_type() {
 function _print_cluster_name() {
 	local test_type="${1:-k8s}"
 	local short_sha
+	local metadata
 
 	if [ -n "${AKS_NAME:-}" ]; then
 		echo "$AKS_NAME"
 	else
 		short_sha="$(git rev-parse --short=12 HEAD)"
-		echo "${test_type}-${GH_PR_NUMBER}-${short_sha}-${KATA_HYPERVISOR}-${KATA_HOST_OS}-amd64-${K8S_TEST_HOST_TYPE:0:1}-${GENPOLICY_PULL_METHOD:0:1}"
+		metadata="${GH_PR_NUMBER}-${short_sha}-${KATA_HYPERVISOR}-${KATA_HOST_OS}-amd64-${K8S_TEST_HOST_TYPE:0:1}-${GENPOLICY_PULL_METHOD:0:1}"
+		# Compute the SHA1 digest of the metadata part to keep the name less
+		# than the limit of 63 chars of AKS
+		echo "${test_type}-$(sha1sum <<< "$metadata" | cut -d'-' -f1)"
```

Comment:
Nit: I would still include at least the PR number in cleartext so it's directly obvious when looking at the list of clusters in the UI and such (may want to truncate the SHA1).

Reply:
I thought about that too, @sprt. Then it will need to truncate the sha1 to keep it within the limit.

Reply:
@wainersm Any issue with addressing this?

Reply:
Ping here :)